packages/app/src/TeamPage.tsx

```diff
@@ -693,6 +801,18 @@ export default function TeamPage() {
                   📋 Copy URL
                 </Button>
               </CopyToClipboard>
+              <Button
```

Comment:
nit: we're trying to move all our buttons to be based on mantine instead of bootstrap ([exmaple](https://github.com/hyperdxio/hyperdx/blob/a2c257f5eb0288af2b97be27269d5642763e9a12/packages/app/src/SaveSearchModal.tsx#L86))

Reply:
Thanks for the review, changes made